Formula/skim.rb

```diff
@@ -0,0 +1,32 @@
+class Skim < Formula
+  desc "Fuzzy Finder in rust!"
+  homepage "https://github.com/lotabout/skim"
+  url "https://github.com/lotabout/skim/archive/v0.5.2.tar.gz"
+  sha256 "41280bee2138afefff95f76a640b753d1cb9215e8391ef37eab7ccb3517d9019"
+
+  depends_on "rust" => :build
+
+  def install
+    (buildpath/"src/github.com/lotabout").mkpath
+    ln_s buildpath, buildpath/"src/github.com/lotabout/skim"
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    prefix.install "install"
+    (prefix/"shell").install %w[bash zsh].map { |s| "shell/key-bindings.#{s}" }
+    (prefix/"shell").install %w[bash zsh].map { |s| "shell/completion.#{s}" }
+    (prefix/"plugin").install "plugin/skim.vim"
+    bin.install "bin/sk-tmux"
+  end
+
+  def caveats; <<~EOS
+    To install useful keybindings and fuzzy completion:
+      #{opt_prefix}/install
+    To use skim in Vim, add the following line to your .vimrc:
+      set rtp+=#{opt_prefix}
+  EOS
+  end
+
+  test do
+    system "#{bin}/sk", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
@fxcoudert `skim` is an interactive tool, does brew have some way to simulate user inputs?

Reply:
https://github.com/Homebrew/homebrew-core/blob/8d0d13e425ea70d42d9597cd90e38e8949150329/Formula/interactive-rebase-tool.rb#L21-L49